src/startup.py

```diff
@@ -14,9 +14,9 @@ def __init__(self):
     def choose_role(self):
         obj_data = self.tag.detect()
         if not obj_data:
-            system('python leader.py')
+            leader.main()
```

Comment:
This is better but I'd prefer actual logic to be here. I don't think this even works? Leader does not have a method called main. There is a method called main in leader.py, but it isn't part of the class.